integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/SessionMigrationUtil.java

```diff
@@ -58,7 +58,7 @@ public static void shutdownServerAndVerify(String domainUid,
     checkPodDoesNotExist(podName, domainUid, domainNamespace);
```

Comment:
Instead of hard-coded sleep time, add a timed wait method as we use in other usecase.

Reply:
I thought about it. However, unfortunately, here we don't have a condition to check. What we are waiting is that WLS picks up a new primary server after the old primary dead but the curl command to query the new primary times out and that's what this change tries to fix

Reply:
Let the condition be "pod does not exist".  Wait till the pod is deleted from the namespace in specified amount of time

Reply:
the test does it at line 58 -     checkPodDoesNotExist(podName, domainUid, domainNamespace);

Reply:
checkPodDoesNotExist return void, so org.awaitility.Awaitility.with can't be applied, unfortunately

Reply:
checkPodDoesNotExist has already a time wait.  So you should add a timed wait during your curl command which check the session migration from primary to secondary

Reply:
changed